Makefile

```diff
@@ -29,10 +30,10 @@ VERSION_MAJOR = $(shell cat $(VERSION_FILE) | cut -f1 -d.)
 VERSION_MINOR = $(shell cat $(VERSION_FILE) | cut -f2 -d.)
 VERSION_BUILD = $(shell cat $(VERSION_FILE) | cut -f3 -d.)
 
-# If your compiler is a bit older you may need to change -std=c++11 to -std=c++0x
+# If your compiler is a bit older you may need to change -std=c++17 to -std=c++0x
 #-I/usr/include/python2.7 -L/usr/lib/python2.7 # -O3
 LLDBFLAGS =
-CXXFLAGS = -c -std=c++11 -m64 -fPIC -I"./btrack/include" \
+CXXFLAGS = -c -std=c++17 -m64 -fPIC -I"./btrack/include" \
 					 -DDEBUG=false -DVERSION_MAJOR=$(VERSION_MAJOR) \
 					 -DVERSION_MINOR=$(VERSION_MINOR) -DVERSION_BUILD=$(VERSION_BUILD) \
 					 -DBUILD_SHARED_LIB
```

Comment:
Looks like the formatting is off here

Reply:
It's the same as before but have changed